### DISCUSSION TITLE
Set Entity Category for controls

### DISCUSSION THREAD

**Discussion by muxa** [fwestenberg/reolink_dev#405] [Ideas]
Home Assistant 2021.11 added support for [entity categories](https://www.home-assistant.io/blog/2021/11/03/release-202111/#entity-categorization)
I think marking all controls (such as FTP, IR Lights, etc) as `config` will be logical and it will remove some noise from the automatic dashboard.

**Comment by cpainchaud** [comment]
Hi, this is interesting as all switches are currently showing as 'config' in my dashboard.... anyways I will force them as config in next release
